notebooks/import.py

```diff
@@ -42,7 +46,20 @@ def get_database_connection():
     return conn, conn.cursor()
 
 
-def create_dataset(exp):
+def create_dataset(exp, date):
+    # Check last import date to avoid importing stale data via backfills.
+    sql = 'SELECT date FROM api_dataset WHERE name=%s'
+    params = [exp]
+    cursor.execute(sql, params)
+    result = cursor.fetchone()
+    if result:
+        prior_import_date = result[0]
+        process_date = datetime.datetime.strptime(date, '%Y%m%d').date()
+        if process_date <= prior_import_date:
+            raise StaleImportError(
+                'Process date %s is older than previous import date %s' % (
+                    process_date, prior_import_date))
+
```

Comment:
Don't we also need to change what we're passing into the `date` column below to be the `date` var?